src/toil/test/__init__.py

```diff
@@ -47,7 +47,7 @@
 from urllib.request import urlopen
 
 
-if sys.version_info >= (3, 8):
+if sys.version_info >= (3, 9):
```

Comment:
My apologies for this small mess. `Literal` is in Python 3.8, the minimum supported version. So it can be pulled out of this check and the matching import from `typing_extensions` removed